phoneApp/js/main.js

```diff
@@ -0,0 +1,119 @@
+const users = [
+    {
+        name: 'Иван',
+        lastName: 'Петров',
+        email: 'ivanpetrov@gmail.com'
+    },
+    {
+        name: 'Сергей',
+        lastName:'Сергеев',
+        email: 'sergeysergeev@gmail.com'
+    },
+    {
+        name: 'Иван',
+        lastName: 'Петров',
+        email: 'ivanpetrov@gmail.com'
+    },
+    {
+        name: 'Александр',
+        lastName: 'Александров',
+        email: 'alex@gmail.com'
+    },
+    {
+        name: 'Алекс',
+        lastName: 'Смирнов',
+        email: 'smirnov@gmail.com'
+    },
+    {
+        name: 'Сергей',
+        lastName: 'Волков',
+        email: 'srgvolk@gmail.com'
+    },
+    {
+        name: 'Мария',
+        lastName: 'Шарапова',
+        email: 'mari@gmail.com'
+    },
+    {
+        name: 'Александр',
+        lastName: 'Винник',
+        email: 'smirnov@gmail.com'
+    },
+];
+const colums = ['Name', 'LastName', 'Email'];
+const app  = {
+    users,
+    colums,
+    render() {
+        document.body.innerHTML += this.renderHeader() +this.renderMain() + this.renderFooter();
+    },
+    renderHeader () {
+        return `
+        <header class="header">
+            <div class="container top-radius">
+                <h2>Contacts</h2>
+        </div>
+        </header>
+        `
+    },
+    renderMain () {
+        return `
+        <main>
+        <div class="container">
+         <form class="form-inline search-form">
+            <div class="form-group">
+                <label class="sr-only" for="search">Search</label>
+                <input type="text" class="form-control" id= "search" placeholder="Search">
+            </div>
+         </form>
+         <table class="table table-hover contacts">
+         <thead>
+          ${this.colums.map(colum => {return`<th>${colum}</th>`}).join('')}
+         </thead>
+         <tbody>
+         ${this.users.map(user => {return`
+             <tr>
+             <td>${user.name}</td>
+             <td>${user.lastName}</td>
+             <td>${user.email}</td>
+             </tr>`
+         }).join('')}</tbody>
+         </table>
+         </div>
+         </main>`
+    },
+    renderFooter () {
+        return `
+        <footer class="footer">
+        <div class="container bottom-radius">
+        <nav class="main-nav">
+            <a href="index.html" class="tab active">
```

Comment:
I believe you could make links more reusable.

What if one day you have to set class but the click ? or something more expensive.

I guess it could look that way
```js
<nav class="main-nav">
 this.renderLink({ content:'Contacts', className:'active', icon:'search'});
 this.renderLink({ content:'Keypad', icon:'th'});
 this.renderLink({ content:'Edit contact' });
 ...
</nav>
```

We could reuse some HTML parts and create a smaller reusable blocks